bindings/node/.evergreen/setup_environment.sh

```diff
@@ -54,14 +56,26 @@ root: $NVM_HOME
 path: $NVM_SYMLINK
 EOT
 
-  nvm install $NODE_VERSION
-  nvm use $NODE_VERSION
+  echo "Running: nvm install lts"
```

Comment:
is it intentional to always use lts for the build instead of the explicit node version set? if so, why are we keeping the defined variable NODE_VERSION in the declarations above?

Reply:
Yea since we don't have a matrix here of node versions (which is fine, our driver will run the actual int tests) I figure the unit tests that run here are pretty safe to always just pick up LTS.
Removed the variable!